bciers/apps/compliance/src/tests/components/compliance-summaries/ComplianceSummariesPage.test.tsx

```diff
@@ -32,8 +35,16 @@ const mockFetchResponse = {
       reporting_year: 2024,
       excess_emissions: 0,
     },
+    {
+      id: 2,
+      operation_name: "Another Test Operation",
+      reporting_year: 2024,
+      excess_emissions: 0,
```

Comment:
There should probably be excess_emissions if there's an obligation that isn't met